scripts/update_appup.escript

```diff
@@ -298,12 +298,14 @@ render_appfile(File, Upgrade, Downgrade) ->
     ok = file:write_file(File, IOList).
 
 create_stub(App) ->
-    case locate(src, App, ".app.src") of
+    case locate(src, App, Ext = ".app.src") of
```

Comment:
nit: maybe make an explicit assignment outside function call to improve readability 🤔